Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -37,12 +37,18 @@ static std::unordered_map<std::string, std::string> s_textureMap;
 static std::unordered_map<std::string, std::shared_ptr<HiresTexture>> s_textureCache;
 static std::mutex s_textureCacheMutex;
 static std::mutex s_textureCacheAquireMutex;  // for high priority access
-static Common::Flag s_textureCacheAbortLoading;
+static Common::Flag s_textureCacheAbortPrefetching;
 static bool s_check_native_format;
 static bool s_check_new_format;
 
 static std::thread s_prefetcher;
 
+static constexpr double CNV_BYT_TO_MB = (1024.0 * 1024.0);
```

Comment:
AFAIK static is not needed here, constexpr on top-level variables have internal linkage.